### ISSUE TITLE
Does stripe connect work with this library

### ISSUE THREAD

**Issue by vikramkh** [tipsi/tipsi-stripe#533]
First of all, thanks for this!

I read #214 and it seems other people want to know as well, does this work with stripe connect?

Also, is there a release coming/should we wait on it, or is it ok to go ahead and start using this without the release?

**Comment by vikramkh** [comment]
So if we use stripe connect, we can't use tipsi stripe then correct? We would have to just use stripe?

**Comment by cybergrind** [comment]
hey @vikramkh basically tipsi-stripe is just a wrapper around tipsi sdk for ios and android. If you can use stripe connect with appropriate mobile SDK => tipsi-stripe also should support that, if you cannot - it won't work.
If everything you need is in #535 - probably it will be added, but this particular PR is missing tests and will require more work

**Comment by augustosamame** [comment]
@cybergrind So it can be done, but how? If you can point me in the right direction I'd appreciate it. I have setup an upwork job to set this up but nobody seems to have any idea of how to do the Stripe Connect integration: https://www.upwork.com/jobs/~01f63293e03129911a

**Comment by bramski** [comment]
New PR: https://github.com/tipsi/tipsi-stripe/pull/563
Rebased against beta.8 including SCA.

**Comment by kilometer1951** [comment]
> @cybergrind So it can be done, but how? If you can point me in the right direction I'd appreciate it. I have setup an upwork job to set this up but nobody seems to have any idea of how to do the Stripe Connect integration: https://www.upwork.com/jobs/~01f63293e03129911a

It's pretty easy. Stripe explains it all in their documentation.

**Comment by guanhuay** [comment]
@vikramkh i just done the same for my project, do you intend to use charges api or payment intent api?

If it is payment intent api, this link will enlight you ===>  https://medium.com/@rafael.rpadovani/stripe-payments-with-sca-in-react-native-15a4926e14f
